refugeerouter/models.py

```diff
@@ -19,15 +19,27 @@ def __str__(self):
 
 
 class Refugee(models.Model):
+    GENDER_FEMALE='F'
+    GENDER_MALE='M'
+    GENDER_DIVERSE='D'
+    GENDER_CHOICES = [
```

Comment:
there is probably also unknown (which should be the default IMHO)

Reply:
fixed